moped-editor/src/auth/user.js

```diff
@@ -73,13 +73,18 @@ export const UserProvider = ({ children }) => {
       return data;
     });
 
+  // Get JWT token with roles
+  const getToken = user => user && user.signInUserSession.idToken.jwtToken;
+
```

Comment:
I love this line! 👍

Reply:
Thanks! I just realized that this function doesn't really need a parameter since `user` is in scope in the context. I'll update and test.